### ISSUE TITLE
Creative tools

### ISSUE THREAD

**Issue by Swisssteel** [B-iggy/HWS-Creative-Scenario#2]
Can the following be added?

1) texture/color tool ability to texture and color multiple blocks at same time?
2) delete block using 'right-click'

Thanks, Swiss

**Comment by B-iggy** [comment]
> 1. texture/color tool ability to texture and color multiple blocks at same time?
> 2. delete block using 'right-click'

I think you just speak of a Creative game. At least this is already possible there.
![image](https://user-images.githubusercontent.com/7815434/102720009-d13e3a80-42f1-11eb-813c-2ed559445d04.png)

You don't have to start a real "Creative game" to do creative stuff.
You can convert any savegame into creative mode by activating it in the console.
![image](https://user-images.githubusercontent.com/7815434/102720022-ee730900-42f1-11eb-9993-bff2a77e0e63.png)

Only for few things you have to relog quick, to trigger the actual mode.
Example if you type "cm" in the console and teleport to a new planet, all POIs will be gone (no POIs in Creative mode).
So switch back to Survival mode with the "cm" command and relog. Then POIs will spawn again.

TL;DR: this is a game mechanic you can activate yourself. Not related to Configs or a Scenario.
